src/svcs/_core.py

```diff
@@ -280,15 +319,32 @@ def register_factory(
                 :class:`collections.abc.Awaitable`; then
                 :meth:`svcs.Registry.aclose()` must be called.
 
+            suppress_context_exit:
+                Whether to suppress errors raised in the container context when
+                exiting registered factories.
+
+                By default, it is True to avoid propagating factory errors to
+                main code. In such case errors are only logged at warning
+                level.
```

Comment:
this one is still confusing to me – it's vice versa, no!?

factory errors never come out, we haven't touched that code AFAICT. This is about passing errors _into_ factories? At least I can't find any tests re that?